tools/scuffed-scraper/requirements.txt

```diff
@@ -0,0 +1,4 @@
+requests==2.31.0
```

Comment:
I would refrain from using hardlimits. If there is a lower limit use that one. That would be preferable.

Reply:
Sure? `requests` has no SemVer-like guarantees. Not locking it might break the script in a few years.

Reply:
So be it :)

Reply:
Just for reference, I added `~=` instead. That should still be robust over time.